### ISSUE TITLE
Compiler is broken (9 errors)

### ISSUE THREAD

**Issue by NeilFraser** [google/blockly#2748]
```

../../core/inject.js:235: ERROR - Access to package-private property setCursor of Blockly.Navigation not allowed here.
  Blockly.Navigation.setCursor(mainWorkspace.cursor);
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/inject.js:236: ERROR - Access to package-private property setMarker of Blockly.Navigation not allowed here.
  Blockly.Navigation.setMarker(mainWorkspace.marker);
  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/keyboard_nav/ast_node.js:139: ERROR - Access to package-private property getParentInput of Blockly.Connection not allowed here.
    return Blockly.ASTNode.createInputNode(connection.getParentInput());
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/keyboard_nav/ast_node.js:141: ERROR - Access to package-private property getParentInput of Blockly.Connection not allowed here.
      && connection.getParentInput()) {
         ^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/keyboard_nav/ast_node.js:142: ERROR - Access to package-private property getParentInput of Blockly.Connection not allowed here.
    return Blockly.ASTNode.createInputNode(connection.getParentInput());
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/keyboard_nav/ast_node.js:480: ERROR - Access to package-private property getParentInput of Blockly.Connection not allowed here.
        topConnection.targetConnection.getParentInput()) {
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/keyboard_nav/ast_node.js:482: ERROR - Access to package-private property getParentInput of Blockly.Connection not allowed here.
        topConnection.targetConnection.getParentInput());
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

../../core/renderers/block_rendering_rewrite/block_render_info.js:67: ERROR - Access to package-private property isInsertionMarker of Blockly.BlockSvg not allowed here.
  this.isInsertionMarker = block.isInsertionMarker();
                           ^^^^^^^^^^^^^^^^^^^^^^^

../../core/renderers/block_rendering_rewrite/measurables.js:195: ERROR - Access to package-private property getCorrectedSize of Blockly.Field not allowed here.
  var size = this.field.getCorrectedSize();
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by alschmiedt** [comment]
This has to do with having the keyboard_nav items in a different folder. Going to pull in my change from render/collab to fix.

**Comment by RoboErikG** [comment]
@alschmiedt is this on master too or just develop?

**Comment by alschmiedt** [comment]
None of my keyboard_nav changes are in master yet so I think it should just be develop.

**Comment by alschmiedt** [comment]
Fixed with #2759
